tests/pkg/lwip/README.md

```diff
@@ -0,0 +1,139 @@
+# lwIP test application
+
+This application provides an example of how to use lwIP and provides test for over-the-air
+interaction between two lwIP nodes.
+
+## Testing on `native`
+
+### IPv6
+
+For IPv6 on `native` there is a test script. It requires for a TAP bridge to be set up between two
+interfaces. For that run
+
+```console
+$ sudo dist/tools/tapsetup/tapsetup
+```
+
+Then just run it with the usual test command.
+
+```
+$ make flash test
+```
+
+### IPv4
+
+For IPv4 a DHCP server is needed. If not already done for the IPv6 tests, please set up a TAP bridge
+now. Then add an IPv4 address to it, e.g.,
+
+```console
+$ sudo ip addr add 10.233.68.1/24 dev tapbr0
+```
+
+Then configure a DHCP server (e.g. [`dhcpd`](https://wiki.archlinux.org/title/Dhcpd))
+
+```
+# comment in if you need DNS and have a name server available
+# option domain-name-servers X.X.X.X X.X.X.Y;
+option subnet-mask 255.255.255.0;
+option routers 10.233.68.1;
+subnet 10.233.68.0 netmask 255.255.255.0 {
+  range 10.233.68.166 10.233.68.206;
+}
+```
+
+Then, you can use the `ip`, `udp`, and `tcp` commands to start servers and connect to them from
+clients.
+
+We use `LWIP_IPV6=0` in the following, but dual-stack support, with both `LWIP_IPV4=1` and
+`LWIP_IPV6=1` is also possible.
+
+## 6LoWPAN
+
+If `<board>` is supported, just running the following should enable 6LoWPAN.
+
+```console
+$ BOARD="<board>" make flash test
+```
```

Comment:
Can we also add

```make
# Instead of simulating an Ethernet connection, we can also simulate
# an IEEE 802.15.4 radio using ZEP
USE_ZEP ?= 0

# set the ZEP port for native
ZEP_PORT_BASE ?= 17754
ifeq (1,$(USE_ZEP))
  TERMFLAGS += -z [::1]:$(ZEP_PORT_BASE)
  USEMODULE += socket_zep

  ifneq (,$(ZEP_MAC))
    TERMFLAGS += --eui64=$(ZEP_MAC)
  endif
endif
```

to the `Makefile` so we can just set `USE_ZEP=1` for 6LoWPAN tests on `native`

Reply:
You can try, but let's do it in a separate PR :-).